### DISCUSSION TITLE
Pluto copy-paste

### DISCUSSION THREAD

**Discussion by gianmariomanca** [fonsp/Pluto.jl#1674] [Pluto development] [answered]
Could it be possible to facilitate copy-paste from one Pluto notebook to another Pluto notebook? Basically selecting cells with the mouse or similar means, copying them and then pasting them into another Pluto notebook?
Now it's a bit clunky if you need to copy many cells, since you have to add the cell in the new notebook, and copy paste the text, one cell at a time.

Is there already a better way?

PS
Some story for deleting a bunch of cells, doing it one by one is a bit cumbersome.
If one could select many cells and then delete them at once it would be much easier.

**Comment by disberd** [comment, marked as answer]
You can already do that.
If you select multiple cells with the mouse drag (started from outside of a cell) and press Ctrl+C (or I suppose Cmd+C on Mac) it copies the code of all selected cells (together with the cell delimiters as they appear on the file) and there is a special paste event on the notebook so that if you paste this on another notebook they are appended to the end of it (or if you paste within a cell in any point in the notebook, it pastes them at that position):
![3be2eb90-d0ba-43d4-a8fd-239f0492a46d](https://user-images.githubusercontent.com/12846528/142379905-1aa8170c-478a-4cc2-9b14-bcddd986dacd.gif)

Edit: You can alse delete all selected cells by pressing canc/backspace after the selection

**Reply by gianmariomanca** [reply]
Great, thank you! Command-C / Command-V seem to do the trick on Mac.
